README.md

```diff
@@ -31,22 +31,22 @@ using the System Properties menu (on Windows).
 
```

Comment:
BTW on th section on setting up environment variables. For me using set var=xx does not work in mac/linux I have to use export var=xx

Reply:
Fixed in latest commits, thanks